app/src/main/java/com/cooper/wheellog/WheelData.java

```diff
@@ -827,6 +826,14 @@ String getAlert() {
         return nAlert;
     }
 
+    public void setAlert(String value) {
+        if (mAlert == "") {
+            mAlert = value;
+        } else {
+            mAlert = mAlert + " | " + mAlert;
```

Comment:
```suggestion
            mAlert += " | " + mAlert;
```
кстати зачем это? Лучше сделать hashset иначе тормоза будут неизбежны. Операция не дешевая для больших строк.

Reply:
Алерт это непредсказуемое и не запрашиваемое событие от колеса. Т.е. может поступить в любой момент. Эта штука нужна только на случай когда второй алерт пришел, а первый еще не забрался в логи. Когда оно считывается через getAlert - оно обнуляется.
В общем пока я даже не видел такого чтобы оно не успело, и в логах чтобы был двойной алерт. Но ситуация в целом не исключена.